docs/hardware.md

```diff
@@ -0,0 +1,48 @@
+# Hardware provisioning
+
+The ROS buildfarm can be provisioned on physical or virtual hardware.
+Because the buildfarm itself utilizes [Docker](https://docker.com) it's not currently possible to run the buildfarm deployment itself in Docker or LXD containers.
```

Comment:
Is it possible to do so with virtual machines? Can we mention any solution for this?
Also, would it be possible for testing purposes to use LXC in privilege mode? https://gist.github.com/kuanghan/9aa5dfea243ed109c0878267e2d80b13